### ISSUE TITLE
Update translations documentation

### ISSUE THREAD

**Issue by Glandos** [spiral-project/ihatemoney#336]
Here is the actual workflow:
```sh
cd ihatemoney
$VENV/bin/pybabel extract --mapping-file babel.cfg -o messages.pot .
$VENV/bin/pybabel update -i messages.pot -d translations/
# Translate your PO file using your favorite editor.
$VENV/bin/pybabel compile -d translations/
```

This should be in the doc and the `Makefile`. Note however that the current POT is **very** outdated. I've run it on a local branch, and nearly every reference has to be updated.

**Comment by 0livd** [comment]
Actually there is already a helper for this in the Makefile which can be run with the following command:

    make build-translations 
But you're right, this should be documented somewhere.

**Comment by Glandos** [comment]
`make build-translations` only compile existing translations to MO files. If new translations string are added (like in #335), then the whole process is needed.

And after running every command, I concluded that this hasn't been done for a loooong time :wink:

**Comment by JocelynDelalande** [comment]
Fixed by #338

**Comment by JocelynDelalande** [comment]
Oh no, this is not fixed as this is not documented nor added to Makefile yet. My bad.
